### DISCUSSION TITLE
Proposals for documentation improvements

### DISCUSSION THREAD

**Discussion by jcorporation** [jcorporation/myMPD#1348] [Ideas]
I plan to rework the documentation site for myMPD. Now is the best time for proposals from your side. Which features should I add and so on. Do you miss a specific topic?

**Comment by john-** [comment]
I would like to see a section on creating and working with playlists.  Maybe I am missing it.

There is a section on smart playlists but I haven't seen anything on non-smart playlists.

**Reply by jcorporation** [reply]
I thought normal playlists are pretty self-explanatory. What do you miss exactly?

**Reply by john-** [reply]
So in fact playlists are self explanatory when you start with the context options  for a song or album.  I just went to confirm and pretty much anywhere a song or album is shown there is an option to add it to new or existing playlist.

The only confusion I have is my first thought is to start by creating a playlist and adding to it.  So I go to Browse Playlists.  The add option there is only to create smart playlists and not regular playlists.  I get why these there are two different approaches for creating these two kinds of playlists.

So after working through writing this I understand it enough to not need a documentation update :)

**Reply by jcorporation** [reply]
MPD can not create an empty playlist. That's why there is no option for this.

**Comment by einarpersson** [comment]
Hi! New user here. 

On https://jcorporation.github.io/myMPD/060-references/smart-playlists/ I miss one thing: 

**One sentence explaining what "smart playlist" is.** 

What is a smart playlist? I see it mentioned as a feature, and the link goes to the page above. It explains how to set it up but not what it is. Sorry if I'm slow!

**Reply by einarpersson** [reply]
And as a general reply to "what can be improved", I would say that for me it would be useful with a kind of POST-installation getting started guide. Something like "you may want to customize THIS, and then perhaps look at THAT... THESE are common things people like to do..." The docs currently lists everything but it's hard to discern the stuff that you actually may want to customize right of the bat, and what is for later.

**Reply by jcorporation** [reply]
Pull requests to improve documentation are very welcome.
